### PR TITLE
Extra scripts & updated versions

### PR DESCRIPTION
**Pull request by pdellaert**
Updated versions for existing scripts to better support different versions of VSPK and pyvmomi. Added some new scripts as well.

### CODE REVIEW

**Comment by primalmotion** [comment]
Looks great! I'll review this on monday

**Comment by pdellaert** [comment]
sure, no rush :). Some features could be added/improved, but it's a start ;)

File: events_overview.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    try:
+        # Connecting to Nuage 
+        try:
+            logger.info('Connecting to Nuage server %s:%s with username %s' % (nuage_host, nuage_port, nuage_username))
+            nc = vsdk.NUVSDSession(username=nuage_username, password=nuage_password, enterprise=nuage_enterprise, api_url="https://%s:%s" % (nuage_host, nuage_port))
+            nc.start()
+        except IOError, e:
+            pass
+
+        if not nc or not nc.is_current_session():
```
Comment (primalmotion) [inline events_overview.py]: Any reasons I miss on why not put that code in the except block?

File: fip_overview.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            'VM MAC': nc_interface.mac, 
+            'FIP': nc_fip.address
+            }
+            json_object.append(json_dict)
+        else:
+            logger.debug('Add row: %s, %s, %s, %s, %s, %s' % (nc_vm.enterprise_name, nc_interface.domain_name, nc_vm.name, nc_interface.ip_address, nc_interface.mac, nc_fip.address))
+            pt.add_row([nc_vm.enterprise_name, nc_interface.domain_name, nc_vm.name, nc_interface.ip_address, nc_interface.mac, nc_fip.address])
+
+    if json_output:
+        print json.dumps(json_object, indent=4)
```
Comment (primalmotion) [inline fip_overview.py]: `sort_keys=True` is a nice `json.dumps` parameter for printing json

**Comment by primalmotion** [comment]
I've added a few little comments, but this looks great. I can't really test assuming the amount of setup I would need to do, but I assume you are using those tools and they work :)

Let me know if you want to take care of the comments, otherwise this is ready to merge

**Comment by pdellaert** [comment]
I'll have a look at those as soon as I am home ;). 

And yes, they are all working with 3.2.R5.

**Comment by pdellaert** [comment]
@primalmotion Thanks for the suggestions, i made some changes to simplify those settings. The exception stuff actually came from something i tend to do when connecting to the vsphere API, but are not necessary with Nuage (should actually check if it is still needed in pyvmomi, might have changed in the latest version).

The JSON sort_keys=True is something that on my setup was being done by default. But just to be sure, i added it :)

**Comment by primalmotion** [comment]
Merged. Thanks!

**Comment by pdellaert** [comment]
Whatever i can do to help :), Thanks!

**Pull request merged**